lib/omniauth/strategies/tiktok_oauth2.rb

```diff
@@ -8,12 +8,12 @@
 module OmniAuth
   module Strategies
     class TiktokOauth2 < OmniAuth::Strategies::OAuth2
-      USER_INFO_URL = 'https://ads.tiktok.com/open_api/v1.2/user/info/'
+      USER_INFO_URL = 'https://business-api.tiktok.com/open_api/v1.2/user/info/'
       option :name, "tiktok_oauth2"
       option :client_options,
-             site: 'https://ads.tiktok.com',
-             authorize_url: 'https://ads.tiktok.com/marketing_api/auth',
-             token_url: 'https://ads.tiktok.com/open_api/v1.2/oauth2/access_token'
+             site: 'https://business-api.tiktok.com',
+             authorize_url: 'https://business-api.tiktok.com/marketing_api/auth',
```

Comment:
Are you sure the `authorize_url` should be changed? According to the linked documentation

> Important
>
> Marketing API portal URLs, including authorization URLs and API doc links, are not changed.

Reply:
`autorize_url` will stay the same!

Reply:
I ran it locally and it works with `https://business-api.tiktok.com/marketing_api/auth` as the `authorize_url`. However, as the documentation points out that it will not be changed let's not change it. Let's just hope the old one still works after September 30th.